### ISSUE TITLE
Urgent: Broken Hebrew/RTL language sign-in pop-up window layout in Plone 5.0.7/8

### ISSUE THREAD

**Issue by MichaelPN** [plone/Products.CMFPlone#2139]
### BUG/PROBLEM REPORT (OR OTHER COMMON ISSUE)

 What I did:

Trying to Log-In back to newly created site in Hebrew language.

### What I expect to happen:
The Log-In to be completed.
### What actually happened:
The Issue arises after creating a new Plone site in RTL language/ or creating a site translated by PAM in RTL language:

The Issue: Log-in window runs to the right behind the visible area of the screen on normal/large screen/resolutions (it is PROBABLY visible on mobile resolutions)

Steps to reproduce:
1.Create a new Plone site in Hebrew or Arabic (did not test on other RTL`s). 
2. Sign Out. 
3.Try to sign in: 
The Login window doesn't  pop-up. Instead  the whole page area is grayed out. The only resolution to someone who is  informed about the problem is to resize the browser window to a   small/mobile size/resolution ,- then in the process of doing that he can  see the log-in window comes back to the middle from behind the right  border of the page. Another way to reproduce this bug is to create a multilingual site  with P.A.M  with  RTL language, and then to repeat the steps  described above while in RTL language. Tested on two different monitors with same  result. 
System is OpenSuse Leap 42.2(64-bit) 
Tested on: Firefox for OpenSuse Leap  52.2.0 (64-bit) 
Chrome 59.0.3071.115 (Official Build) (64-bit)
### What version of Plone/ Addons I am using:
Plone 5.0.7/8

**Comment by pbauer** [comment]
I can confirm this. It seems nobody tested or properly implemented rtl for Barceloneta. You are welcome to fix this.

**Comment by MichaelPN** [comment]
I don't do any programming or CSS design unfortunately. Sorry.But this thing sure makes Plone 5 almost unusable  for users in  RTL until fixed.

    On Tuesday, September 12, 2017 3:43 PM, Philip Bauer <notifications@github.com> wrote:
 

 I can confirm this. It seems nobody tested or properly implemented rtl for Barceloneta. You are welcome to fix this.—

**Comment by pbauer** [comment]
I see, maybe we can find someone to work on it during the conference-sprint. 
@hvelarde I don't consider that a blocker that should delay Plone 5.1. Plone 5 has been out for 2 years now an noone seems to have cared enough about this issue to have reported it.

**Comment by MichaelPN** [comment]
It  appears that   Plone 5.0b2 doesn't have this issue.Tested on plone5demo.quintagroup.com .


 

    On Saturday, September 16, 2017 12:46 PM, Philip Bauer <notifications@github.com> wrote:
 

 I see, maybe we can find someone to work on it during the conference-sprint.
@hvelarde I don't consider that a blocker that should delay Plone 5.1. Plone 5 has been out for 2 years now an noone seems to have cared enough about this issue to have reported it.—

**Comment by jensens** [comment]
I close the issue, because it addresses a Plone version that is no longer supported. 
If you think this is wrong please reopen the issue and assign a matching milestone.
